lib/sntools.test.js

```diff
@@ -175,6 +175,67 @@ describe('sn-tools', () => {
     });
   });
 
+  describe('convertAegisToSN()', () => {
+    const filePath = join(__dirname, '../test/data/aegis/backup.json');
+    const content = readFileSync(filePath);
+    const metadata = statSync(filePath);
+    const parsedFile = tools.convertAegisFile({
+      name: 'backup.json',
+      lastModified: metadata.mtime,
+      content,
+    });
+
+    it('should return 2 valid items', () => {
+      expect(parsedFile).not.toBe(undefined);
+      expect(parsedFile.items).toBeDefined();
+      expect(parsedFile.items.length).toBe(2); // 1 Note and 1 Tag
+
+      expect(parsedFile).not.toBe(undefined);
+      expect(parsedFile.items).toBeDefined();
+      expect(parsedFile.items.length).toBe(2); // 1 Note and 1 Tag
+    });
+
+    const { items } = parsedFile;
+
+    const firstItem = items[0];
+    const secondItem = items[1];
+    
+    test("first item should be a Tag", () => {
+      expect(firstItem).toEqual({
+        uuid: expect.stringMatching(uuidFormat),
+        content_type: 'Tag',
+        content: {
+          title: '13-3-2022-import',
```

Comment:
This will fail if a different locale is used. Perhaps you can change it to something like:

```suggestion
          title: `${new Date('2022-03-13').toLocaleDateString().replace(/\//g, '-')}-import`,
```